### ISSUE TITLE
 redis compiling error

### ISSUE THREAD

**Issue by shijinkui** [openzipkin/zipkin#266]
![qq20130627-1](https://f.cloud.github.com/assets/648508/715909/028d1d2e-df2a-11e2-972e-cf44b9ced503.png)

i use redis replace cassandra.

compile error.

database.zRevRangeByScore(preface(key), ZInterval(stop), ZInterval(start), true, Some(Limit(0, count)))

this method invoking have no problem, why compile err.

**Comment by mosesn** [comment]
You have an old version of finagle-redis somewhere on your classpath.

**Comment by mosesn** [comment]
Ah this is in master.  I thought someone else had fixed it, I'll hack something together right now.

**Comment by mosesn** [comment]
@shijinkui, @jaredlwong has a PR #253 out to fix this.  It hasn't been merged into master yet, but you can run `git pull git@github.com:jaredlwong/zipkin.git redis` in a branch in your zipkin repository to update it manually.

**Comment by bmdhacks** [comment]
I can't merge it until the tests pass.  Franklin added redis to the travis
setup so it should be just a rebase to fix it.  Can somebody do that and
resubmit?

**Comment by shijinkui** [comment]
@mosesn , compiling passed.  thx

**Comment by bmdhacks** [comment]
This is fixed now
